sugondat-chain/primitives/src/lib.rs

```diff
@@ -5,6 +5,9 @@ use sp_runtime::{
     MultiSignature,
 };
 
+// TODO: probably this could be moved into runtimes/kusama-runtime/src/constants.rs
```

Comment:
I don't have an opinion, but given that it's used across runtimes, this seems like a fine location for it.